src/App.scss

```diff
@@ -1,2 +1,83 @@
 @use "theme/utilities.scss";
 @use "theme/global.scss";
+
+
+.App{
+    background-color: #8ecae6;
+}
+
+.c-site-header{
+  padding: 20px;
+}
+
+.heading{
+  background-color: #219ebc;
+  padding: 10px;
+}
+
+.c-site-header__title{
+  color: #023047;
+  padding: 20px;
+}
+
+.city{
+  margin: 15px;
+  padding: 15px;
+}
+
+.search-btn{
+  background-color: #023047;
+  color: white;
+  margin-left: 15px;
+  padding: 15px;
+}
+
+.weather-img{
+  margin: auto;
+  width: 200px;
+  height: 200px;
+  object-fit: cover;
+}
+
+.description{
+  display: flex;
+  justify-content: center;
+  color: white;
+}
+
+.temp {
+  display: flex;
+  justify-content: center;
+  margin: 20px;
+}
+
+.box{
+  display: flex;
+  justify-content: center;
+}
+
+.sub-box{
+  padding: 30px;
+}
+
+
+// -----------future weather------
+
+.container{
+ display: flex;
+ width: 100%;
+ padding: 0px 2em 0px 2em;
+}
+
+.items{
+  display: flex;
+  background-color: #023047;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  border: 1px solid #fb8500;
```

Comment:
If you added this border to help you in your development that's fine, but don't commit it 😉

Reply:
Sure.It's deleted now.